cpp/ps0005/problem solve 5.cpp

```diff
@@ -1,53 +1,56 @@
 #include <iostream>
 using namespace std;
-
-int main() 
-{ 
-    float num1 , num2;
+int main() {
+    // prompting user for 2 operands and an operator 2/2
+    float num1, num2;
     char operation;
     cout << "Enter first number:\n";
     cin >> num1;
-    cout << "Enter operation :\n";
+    cout << "Enter operation (+, -, *, /, % , ^):\n";
     cin >> operation;
     cout << "Enter second number:\n";
     cin >> num2;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input